pkg/apihelpers/apihelpers.go

```diff
@@ -8,8 +8,73 @@ import (
 	"fmt"
 
 	mcfgv1 "github.com/openshift/api/machineconfiguration/v1"
+	opv1 "github.com/openshift/api/operator/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog/v2"
+)
+
+var (
```

Comment:
I'm fine with it here but out of curiosity why did we decide to define this in apihelpers?

Reply:
I sort of interpreted this as our internal MCO API specific helpers. I guess I could've put it in the regular helpers too, it just felt natural to put it there as NodeDisruption is an MCO only API object (: